config/clusters/nasa-veda/common.values.yaml

```
@@ -65,9 +65,9 @@ basehub:
     singleuser:
       defaultUrl: /lab
       image:
-        name: pangeo/pangeo-notebook
+        name: public.ecr.aws/nasa-veda/nasa-veda-singleuser
         # Uses JupyterLab <4, so jupyterlab-git and dask-dashboard work
```

```suggestion
```

This comment refers to the image you are replacing, so we don't need to keep it :)

---

Yuvi included that comment there for the tag "2023.07.05" which [my new base image is still using for the same reason](https://github.com/NASA-IMPACT/veda-jh-environments/blob/main/docker-images/base/pangeo-notebook/Dockerfile#L1) so I say let's keep it for posterity if you don't mind

---

```suggestion
        # Based off pangeo/pangeo-notebook:2023.07.05 which uses JupyterLab <4, so jupyterlab-git and dask-dashboard work
```

No worries! How about this so we're explicit about what is using JupyterLab<4?